x-pack/solutions/security/plugins/security_solution/public/detections/components/attacks/table/table_section.test.tsx

```diff
@@ -377,4 +377,55 @@ describe('<TableSection />', () => {
       });
     });
   });
+
+  describe('enforced groups', () => {
+    it('should include enforcedGroups in groupingSettings', () => {
+      expect(groupingSettings.enforcedGroups).toBeDefined();
+      expect(Array.isArray(groupingSettings.enforcedGroups)).toBe(true);
+    });
```

Comment:
This test feels weird. Verifying that a imported value is what we expect is odd... plus it is already covered by the last test below (`should pass all grouping settings including enforcedGroups`).
I think we should remove it.